Samples/NoFilters.tests.json

```diff
@@ -28,7 +28,7 @@
             "Exception": "Invalid setting 'enabled' with value 'invalid' for feature 'InvalidEnabled'."
         },
         "Variant": {
-            "Result": null
+            "Exception": "Invalid setting 'enabled' with value 'invalid' for feature 'InvalidEnabled'."
```

Comment:
Did you run this test with python? I'm unsure it would produce this exact message.

Reply:
Turns out I was running the tests wrong. Latest commit has them working in .net and python